pkg/gencmd/providers.go

```diff
@@ -0,0 +1,22 @@
+package gencmd
+
+import (
+	"net/http"
+
+	"github.com/google/go-cloud/wire"
+
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+)
+
+func ProvideGrapiCtx(ctx *Ctx) *grapicmd.Ctx         { return ctx.Ctx }
+func ProvideTemplateFS(cmd *Command) http.FileSystem { return cmd.TemplateFS }
+func ProvideShouldRun(cmd *Command) ShouldRunFunc    { return cmd.ShouldRun }
+
+var Set = wire.NewSet(
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var Set should have comment or be unexported